cmds/webboot/webboot_test.go

```diff
@@ -183,6 +183,49 @@ func TestBackOption(t *testing.T) {
 	}
 }
 
+func TestDisplayChecksumPrompt(t *testing.T) {
+	// test data
+	var testDistros = map[string]Distro{
+		"FakeDistro": {
+			checksum:     "1234567",
+			checksumType: "sha256",
+		},
+		"FakeDistroNoChecksum": {},
+		"FakeDistroGoodChecksum": {
+			checksum:     "407dc87b95afbe268e760313971041860f36e953a2116db03418a98ce46d61bc",
+			checksumType: "sha256",
+		},
+	}
+
+	type test struct {
+		keyInput    []string
+		distroName  string
+		want        string
+		description string
+	}
+
+	tests := []test{
+		{keyInput: []string{"1", "<Enter>"}, distroName: "FakeDistro", want: "*main.DownloadOption", description: "Incorrect checksum, don't proceed"},
+		{keyInput: []string{"0", "<Enter>"}, distroName: "FakeDistro", want: "<nil>", description: "Incorrect checksum, proceed"},
+		{keyInput: []string{"1", "<Enter>"}, distroName: "FakeDistroNoChecksum", want: "*main.DownloadOption", description: "No checksum, don't proceed"},
+		{keyInput: []string{"0", "<Enter>"}, distroName: "FakeDistroNoChecksum", want: "<nil>", description: "No checksum, proceed"},
+		{keyInput: []string{}, distroName: "FakeDistroGoodChecksum", want: "<nil>", description: "Correct checksum"},
```

Comment:
Some of these lines are a bit long. They could be shortened before "description:" for easier readability.